lib/Basics/Thread.cpp

```diff
@@ -35,6 +35,10 @@
 #include <chrono>
 #include <thread>
 
+#ifdef TRI_HAVE_PROCESS_H
+#include <process.h>
```

Comment:
I assume this is only required because of the `_getpid` call in line 104. In this case you could use `GetCurrentProcessId` instead (which is defined in the omnipresent Windows.h), so you could ditch the process.h include.

Reply:
well, one of the targets would be to get rid of windows.h as much as we can too. Imho staying with the more posixalike function is no mistake then.

Reply:
I was assuming that `GetCurrentProcessId` is available because Windows.h is implicitly included, but is just a minor note. 😉